commands-yml/commands/element/active.yml

```diff
@@ -0,0 +1,70 @@
+---
+name: Active Element
```

Comment:
"Active Element" doesn't sound like a command to me. maybe "Get Active Element"? or "Find Active Element"?